RaceTrack/Tournament/Tournament.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DataRace;
+
+namespace RaceTournament
+{
+    public class TournamentResult
+    {
+        private Queue<Racer> _racers;
+        public Queue<Racer> _winners = new Queue<Racer>();
+        public Action<Racer, Racer, Racer> OnRaceCompleted;
+
+        public TournamentResult(List<Racer> racers)
+        {
+            _racers = new Queue<Racer>(racers);
+        }
```

Comment:
It's better to distinguish between `Tournament` entity and `TournamentResult`. Please refactor to make classes more logical